src/main/java/io/cdap/plugin/successfactors/common/exception/proto/innererror/ErrorResolution.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.successfactors.common.exception.proto.innererror;
+
+/**
+ * ErrorResolution class
+ */
+public class ErrorResolution {
+
+  private final String sapTransaction;
+  private final String sapNote;
+  private final String additionalSAPNote;
+
+  public ErrorResolution(String sapTransaction, String sapNote, String additionalSAPNote) {
```

Comment:
can we have more successfactors specific name?

Reply:
Fields have been renamed appropriately. 4dbd00c9806f47df7f78cf411834bc8c7a613f94